h2/pom.xml

```diff
@@ -212,6 +212,13 @@
     </testResource>
     </testResources>
     <plugins>
+      <plugin>
+	<artifactId>maven-compiler-plugin</artifactId>
+	<version>3.6.0</version>
+	<configuration>
+	  <useIncrementalCompilation>false</useIncrementalCompilation>
+	</configuration>
+      </plugin>
```

Comment:
It's a maven java compiler setting for incremental compilation.

Reply:
And why you need this setting?

Reply:
This speeds up the build and hence development cycle.

Reply:
Once again, this file is not used and should not be used to build H2 from its sources, it can't do this work properly. It is an unfinished experiment.

Normal building procedure is described here:
https://h2database.com/html/build.html#building

During development process you can simply use your IDE, there is no need to build jar after each change.

Your code doesn't compile, BTW. Take a look on Travis CI logs:
https://travis-ci.org/github/h2database/h2database/builds/668272621

Reply:
Removed this section. Tests are passing now.